.eslintrc.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
     "linebreak-style": ["error", "unix"],
     quotes: ["error", "double", { avoidEscape: true }],
     semi: ["error", "always"],
-    "spaced-comment": ["error", "always", { exceptions: ["-", "+"] }]
+    "spaced-comment": ["error", "always", { exceptions: ["-", "+"] }],
+    "mocha/no-exclusive-tests": "error"
```

Comment:
Will `--fix` remove the `it.only`?

Reply:
nope. --fix does not remove them. it just throws the linter error